x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/expanded_row_messages_pane.tsx

```diff
@@ -110,11 +133,20 @@ export const ExpandedRowMessagesPane: React.FC<Props> = ({ transformId }) => {
         }
       ),
       width: '50%',
+      sortable: true,
```

Comment:
Sorting isn't working for me on the `Message` column. If it's difficult technically, I notice we don't make this column sortable for anomaly detection or DFA jobs.

![image](https://user-images.githubusercontent.com/7405507/175897584-f6537070-5e3d-44da-b0c9-ca7ac6e6b65a.png)

Reply:
Updated here https://github.com/elastic/kibana/pull/135047/commits/24f257a6a6ca53959885a9311171f6cb1b0b12dc